morf-core/src/main/java/org/alfasoftware/morf/jdbc/UrlConnectionResourcesBean.java

```diff
@@ -0,0 +1,247 @@
+/* Copyright 2017 Alfa Financial Software
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.alfasoftware.morf.jdbc;
+
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+import javax.sql.DataSource;
+import javax.sql.XADataSource;
+import javax.xml.crypto.Data;
+
+import org.alfasoftware.morf.metadata.SchemaResource;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import com.google.common.base.Preconditions;
+
+/**
+ * A {@link ConnectionResources} implementation which uses only the JDBC URL,
+ * where possible.
+ * 
+ * <p>Create using
+ * {@link DatabaseType.Registry#urlToConnectionResources(String)}.</p>
+ */
+public class UrlConnectionResourcesBean implements ConnectionResources {
+
+  private static final Log log = LogFactory.getLog(UrlConnectionResourcesBean.class);
+  
+  private final String url;
+  private final String databaseType;
+  private final String databaseName;
+  
+  private String schemaName;
+  private int transactionIsolationLevel = Connection.TRANSACTION_READ_COMMITTED;
```

Comment:
I would expect this to default to "whatever is supplied by the driver", rather than read-committed specifically (which is a bit of an Alfaism)

Reply:
Mmm, could just make it optional.